docs/input-jdbc.asciidoc

```diff
@@ -176,6 +176,29 @@ input {
 }
 ---------------------------------------------------------------------------------------------------
 
+==== Known limitations
+
+Theoretically, this plugin can function with any JDBC driver.
+However, the implementation of drivers may vary, potentially leading to unexpected behavior in connection pool management.
+
+===== Unable to reuse connection
+
+Stale connections may return to the pool, causing the plugin reuse closed connections. Sybase is affected by this issue.
```

Comment:
Some databases - such as Sybase or SQL Anywhere - may have issues with stale connections, timing out between scheduled runs and never reconnecting.